src/ncp/ncp_base.cpp

```diff
@@ -2975,10 +2975,41 @@ ThreadError NcpBase::GetPropertyHandler_THREAD_RLOC16_DEBUG_PASSTHRU(uint8_t hea
 
 ThreadError NcpBase::GetPropertyHandler_THREAD_LOCAL_ROUTES(uint8_t header, spinel_prop_key_t key)
 {
-    // TODO: Implement get external route table
-    (void)key;
+    ThreadError errorCode = kThreadError_None;
+    otExternalRouteConfig external_route_config;
+    otNetworkDataIterator iter = OT_NETWORK_DATA_ITERATOR_INIT;
+    uint8_t flags;
 
-    return SendLastStatus(header, SPINEL_STATUS_UNIMPLEMENTED);
+    mDisableStreamWrite = true;
+
+    SuccessOrExit(errorCode = OutboundFrameBegin());
+    SuccessOrExit(errorCode = OutboundFrameFeedPacked(SPINEL_DATATYPE_COMMAND_PROP_S, header, SPINEL_CMD_PROP_VALUE_IS,
+                                                      key));
+
+    while (otNetDataGetNextRoute(mInstance, false, &iter, &external_route_config) == kThreadError_None)
```

Comment:
Checking `GetPropertyHandler_THREAD_ON_MESH_NETS()` implementation (which is intended to get all prefix entries from network data), there we also get the partition's network data (and not the lcoal one). 
https://github.com/openthread/openthread/blob/master/src/ncp/ncp_base.cpp#L2752